tmexp/cli.py

```diff
@@ -0,0 +1,139 @@
+from argparse import ArgumentParser
+from collections import OrderedDict
+from logging import _nameToLevel as logging_name_to_level
+from typing import Any, Callable, Dict, NamedTuple, Optional, Tuple
+
+from .utils import check_create_default, SUPPORTED_LANGUAGES
+
+
+class CommandDescription(NamedTuple):
+    help: str
+    parser_definer: Callable[[ArgumentParser], None]
+    handler: Callable[..., Any]
+
+
+_commands: Dict[str, CommandDescription] = OrderedDict()
+
+
+def register_command(
+    *,
+    parser_definer: Callable[[ArgumentParser], None],
+    name: Optional[str] = None,
+    help: Optional[str] = None
```

Comment:
I agree. My reasonning was to avoid constraining docstrings too much (and function names) if you wanted to decouple those from command building but if we are on the same page I will remove them :)